index.html

```diff
@@ -53,90 +50,159 @@ <h3>I'm a Software Developer</h3>
+                    <p class="text">Spoiled Eggs was the original template for 2 projects and those two projects were used to promote Spoiled Eggs.  It's a review site for opera, where people can rage click "applause" or "throwing eggs" and of course comment.</p>
+                </div>
+                <ul class="items lang">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>php</li>
+                    <li>mysql</li>
+                    <li>javascript</li>
+                    
+                </ul>
+            </div><!--greyout end-->
+            <div class="buttondecor">
+                <p>See Project</p>
+            </div></a>
         </div><!--site1 end-->
-        <br> <br> <br>
-        <div class="items site1 site2">
-            <div class="description2">
-                <div class="title3">Multi-Post Stories</div>
-                <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-            </div>
-            <ul class="items lang">
-                <li>html</li>
-                <li>css</li>
-                <li>php</li>
-                <li>mysql</li>
-                <li>javascript</li>
-                
-            </ul>
-            <div class="items buttontypeB">
-                <a href="#">See Project</a>
+        <div class="site site2">
+         <a href="#portfolio"><div class="greyout">
+                <div class="description">
+                    <div class="title">Classica Tournée</div>
+                    <p class="text">A site meant as an extension of Opera Classica Europa's Inthega classifieds, which is a platform intended on facilitating touring in Germany.</p>
+                </div>
+                <ul class="items lang">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                    
+                </ul>
+            </div><!--greyout end-->
+            <div class="buttondecor">
+                <p>Not Active</p>
+            </div></a>
+        </div><!--site1 end-->
+        <div class="site site3">
+         <a href="#portfolio"><div class="greyout">
+                <div class="description">
+                    <div class="title">Opera Pathway</div>
+                    <p class="text">The advertisement and application process for Opera Classica Europa's Opera Studio program, called the Opera Pathway.</p>
+                </div>
+                <ul class="items lang">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                    
+                </ul>
+            </div><!--greyout end-->
+            <div class="buttondecor">
+                <p>Not Active</p>
+            </div></a>
+        </div><!--site1 end-->
```

Comment:
It should be good if you remove the commented code in your work, this is to help you to keep your code cleaned